.github/workflows/preview.yml

```diff
@@ -16,16 +16,49 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.repository == 'growthbook/growthbook' }}
     steps:
-    - name: Context
-      uses: okteto/context@latest
-      with:
-        url: ${{ secrets.OKTETO_URL }}
-        token: ${{ secrets.OKTETO_TOKEN }}
+      - name: Context
+        uses: okteto/context@latest
+        with:
+          url: ${{ secrets.OKTETO_URL }}
+          token: ${{ secrets.OKTETO_TOKEN }}
 
-    - name: Deploy preview environment
-      uses: okteto/deploy-preview@latest
-      env:
-       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        name: pr-${{ github.event.number }}-bttf
-        timeout: 30m
+      - name: Deploy preview environment
+        id: deploy-preview
+        uses: okteto/deploy-preview@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: pr-${{ github.event.number }}-bttf
+          timeout: 30m
+
+  stably-test:
+    needs: [preview]
+    name: Stably Test Runner
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+
+      - name: Stably Runner Action
+        id: stably-runner
+        uses: stablyhq/stably-runner-action@v3
+        with:
+          api-key: ${{ secrets.STABLY_API_KEY }}
+          run-group-ids: |-
+            clp2xrhh60001l608s1b4w4zd
```

Comment:
I created a new "Test Group" so we can control exactly what tests run via this GH action. Currently, there is only a single test, but we can update this suite without needing to update the GH action.

Reply:
Currently, we're running into a timeout issue with a lot of our main tests - Stably is addressing that today.